cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/ZooKeeperRoller.java

```diff
@@ -128,14 +135,76 @@ public Future<Void> maybeRollingUpdate(Reconciliation reconciliation, Labels sel
      * @return a Future which completes when the given (possibly recreated) pod is ready.
      */
     Future<Void> restartPod(Reconciliation reconciliation, String podName, List<String> reasons) {
-        long pollingIntervalMs = 1_000;
-
         LOGGER.infoCr(reconciliation, "Rolling pod {} due to {}", podName, reasons);
         return podOperator.getAsync(reconciliation.namespace(), podName)
                 .compose(pod -> podOperator.restart(reconciliation, pod, operationTimeoutMs))
                 .compose(ignore -> {
                     LOGGER.debugCr(reconciliation, "Waiting for readiness of pod {}", podName);
-                    return podOperator.readiness(reconciliation, reconciliation.namespace(), podName, pollingIntervalMs, operationTimeoutMs);
+                    return podOperator.readiness(reconciliation, reconciliation.namespace(), podName, READINESS_POLLING_INTERVAL_MS, operationTimeoutMs);
                 });
     }
+
+
+    private static class ZookeeperClusterRollContext {
```

Comment:
I'm not seeing anything `Zookeeper` specific about the context classes, I wonder if it would be better to remove the prefix and inflate them to top level classes?

Reply:
Removing the prefix sounds good, but I'd hesitate on pulling them up given it's a first cut. Best to keep them trapped in here at first. I don't want to help the `List<String>` of restart reasons to proliferate. The `KafkaRoller` is working with the `RestartReasons` class which would be better.

Reply:
Ok Sounds reasonable

Reply:
I think there is no real value from separating them right now. So I would keep it inside this class.